docs/dev_docs/fetcher.md

```diff
@@ -1,6 +1,6 @@
 # Fetcher
 
-The fetcher is used to pull whitelisted files/folders from one or more git repositories. It takes a string containing 3 values:  
+The fetcher is used to pull whitelisted files/folders from one or more git repositories. The progress of the fetcher can be tracked using the small spinner and progress bar displayed in the terminal. It takes a string containing 3 values:  
 1. A URL to a git based repository
 2. A branch name
 3. A path to a whitelist file  
```

Comment:
```suggestion
1. An URL to a git-based repository
2. A branch name
3. A path to a whitelisted file
```

Reply:
It is actually 🤓 "A URL" not "An URL"

Reply:
Um actually, I concede 🤓

Reply:
3. is wrong. It is a path to a whitelist file. The whitelist file contains whitelisted files or folders